deps/sandbox.c

```diff
@@ -480,6 +500,8 @@ static int sandbox_main(const char * root_dir, const char * new_cd, int sandbox_
         // If it was the main pid that exited, return as well.
         return WIFEXITED(status) ? WEXITSTATUS(status) : 1;
       }
+      // Sleep for 100ms so that we're not chewing through CPU cycles
+      usleep(100000);
```

Comment:
:zzz:

Reply:
We should just stop passing WNOHANG to waitpid

Reply:
Yeah, duh.  For some reason I thought we were doing more work within this `while` loop.....